plugins/NSFW.js

```diff
@@ -1185,14 +1189,31 @@ function holdHandler(damageHP, damageLust, isSexual, actionTier) {
 
         currentFight.currentHold = {
             holdName: holdName,
+            holdType: undefined,
             turnsLeft: newTurnsLeft,
             damageHP: newDamageHP,
             damageLust: newDamageLust,
             attacker: attacker,
             defender: defender,
-            isInfinite: isInfinite
+            isInfinite: isInfinite,
+            actionTier: actionTier
```

Comment:
This may cause a merge conflict with #53, but they both accomplish the same thing.
Copy and pasting is bad for the soul.